__tests__/puppeteer-tabbed-editor.test.js

```diff
@@ -8,10 +8,10 @@ describe("Tabbed Editor", () => {
       const expectedCSS =
         " address { font-variant-caps: small-caps;}a { font-variant: normal;} ";
       const expectedHTML =
-        "<address> James Rockford<br> 2354 Pacific Coast Highway<br> " +
+        " <address> James Rockford<br> 2354 Pacific Coast Highway<br> " +
         "California<br> USA<br> +311-555-2368<br> Email: " +
         '<a href="mailto:j.rockford@example.com">j.rockford@example.com</a><br>' +
-        "</address>";
+        "</address> ";
```

Comment:
why this change?

Reply:
It's because prettier has wrapped #html-output element into multiple lines, creating whitespace. The same thing happens with `expectedCSS`, so I chose to add space to `expectedHTML` instead of using trim.